src/D2L.CodeStyle.Analyzers/Async/Generator/AsyncToSyncMethodTransformer.cs

```diff
@@ -392,6 +396,14 @@ private VariableDesignationSyntax Transform( VariableDesignationSyntax des )
 		);
 	}
 
+	private SeparatedSyntaxList<TypeSyntax> TransformTypes( SeparatedSyntaxList<TypeSyntax> typeDecls ) {
+		return SyntaxFactory.SeparatedList(
+			typeDecls.Select( typeDecl =>
+				TransformType( typeDecl )
+			)

```

Comment:
nit, I think this is sufficient?

```suggestion
			typeDecls.Select( TransformType )
```

Reply:
(To be clear these are usually equivalent, just less verbose.)